index.js

```diff
@@ -45,34 +45,6 @@ app.all('*', (req, res) => res.status(404).json({
   error: 'Page not found.',
 }));
 
-// / error handlers
```

Comment:
@leksyib why are we taking this out?

Reply:
@9jaswag  we're not using them. We're handling those errors in try/catch block of our controllers.